getting-started/gce/deploy.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/sh
+
+# Copyright 2019 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+if [ -z "$DEPLOY_DIR" ]; then
+  echo "Must set \$DEPLOY_DIR. For example: DEPLOY_DIR=gs://my-bucket"
+  exit 1
+fi
+
+if [ -z "$DEPLOY_FILENAME" ]; then
+  echo "Must set \$DEPLOY_FILENAME. For example: DEPLOY_FILENAME=app.tar.gz"
+  exit 1
+fi
```

Comment:
Maybe these should be positional arguments? People seem to have problems with environment variables.

But maybe that's more about Windows and we're already really excluding them if they're not using WSL.

Reply:
I don't think we'll be telling users to run this script in the tutorial, so I'm going to leave as-is for now. But, happy to change if it ends up being more clear.

Agreed about Windows. Maybe we'll add Powershell scripts later.